CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 #### Upcoming Changes
 
+* feat: [#1838](https://github.com/lambdaclass/cairo-vm/pull/1824):
```

Comment:
@JulianGCalderon Change this, use just one line to describe that you add the dynamic layout support and link the two PRs